scene/__init__.py

```diff
@@ -81,9 +81,20 @@ def __init__(self, args : ModelParams, gaussians : GaussianModel, load_iteration
                                                            "iteration_" + str(self.loaded_iter),
                                                            "point_cloud.ply"))
             self.gaussians._vdgs = load(os.path.join(self.model_path, "point_cloud", "iteration_" + str(self.loaded_iter), "mlp_model.pt"))
+            with open(os.path.join(self.model_path, "vdgs_settings.json"), 'r') as file:
```

Comment:
In this part the `self.gaussians.vdgs_type` and `self.gaussians.vdgs_type` are still None.